src/ColorVectorSpace.jl

```diff
@@ -245,42 +261,35 @@ min(a::AbstractGray, b::AbstractGray) = min(promote(a,b)...)
 min(a::Number, b::AbstractGray) = min(promote(a,b)...)
 min(a::AbstractGray, b::Number) = min(promote(a,b)...)
 
-norm(c::AbstractGray) = abs(gray(c))
-abs(c::TransparentGray) = abs(gray(c))+abs(alpha(c)) # should this have a different name?
-abs(c::TransparentGrayNormed) = Float32(gray(c)) + Float32(alpha(c)) # should this have a different name?
-abs2(c::AbstractGray) = gray(c)^2
-abs2(c::AbstractGray{T}) where {T<:Normed} = Float32(gray(c))^2
-abs2(c::TransparentGray) = gray(c)^2+alpha(c)^2
-abs2(c::TransparentGrayNormed) = Float32(gray(c))^2 + Float32(alpha(c))^2
-atan(x::Gray, y::Gray) = atan(convert(Real, x), convert(Real, y))
-hypot(x::Gray, y::Gray) = hypot(convert(Real, x), convert(Real, y))
-norm(c::TransparentGray) = sqrt(abs2(c))
-
-(<)(g1::AbstractGray, g2::AbstractGray) = gray(g1) < gray(g2)
-(<)(c::AbstractGray, r::Real) = gray(c) < r
-(<)(r::Real, c::AbstractGray) = r < gray(c)
+atan(x::AbstractGray, y::AbstractGray)  = atan(gray(x), gray(y))
+hypot(x::AbstractGray, y::AbstractGray) = hypot(gray(x), gray(y))
+
+if !hasmethod(<, Tuple{AbstractGray,AbstractGray})  # this was moved to ColorTypes 0.11
```

Comment:
Should we continue to support the old (current) FixedPointNumbers and ColorTypes?
This version should be a "new generation", so I think it's better to break the dependency, even if they are technically compatible.

Reply:
That would be fine. This is easier for me locally because I switch between `dev`ved and `free`d versions of packages, but we could hardwire this in the final version.